src/utils.coffee

```diff
@@ -177,10 +177,29 @@ exports.resinLogControl = (val) ->
 	logger.disableLogPublishing(val == 'false')
 	console.log('Logs enabled: ' + val)
 
+gosuperRequest = (method, endpoint, options = {}, callback) ->
+	if _.isFunction(options)
+		callback = options
+		options = {}
+	options.method = method
+	options.url = config.gosuperAddress + endpoint
+	options.headers ?= {}
+	options.headers['Host'] = ''
```

Comment:
Changed it  to use it @Page- 
I think we can probably use it better (e.g. have gosuperPost and gosuperGet be defaults on top of this), but it feels we don't win much and the result might be a bit convoluted.